python/lsst/ts/standardscripts/maintel/laser_tracker/shut_down.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["ShutDown"]
+
+import asyncio
+
+from lsst.ts import salobj
+
+
+class SetUp(salobj.BaseScript):
+    """Shut down Laser Tracker.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+    - "Shutting down Laser Tracker.": Turning laser power off.
+    - "Laser Tracker turned off.": Laser Tracker turned off.
+    """
+
+    def __init__(self, index):
+        super().__init__(index, descr="Shut down laser tracker.")
+
+        self.config = None
+
+        self.laser_tracker = salobj.RemoteGroup(
+            domain=self.domain, components=["LaserTracker:1"], log=self.log
+        )
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        self.config = config
+
+    async def run(self):
+        """Run the script.
+        Raises
+        ------
+        asyncio.TimeoutError
+            If the Laser Tracker is not ready after the timeout.
+        """
+        await self.start_up()
+
+        await self.checkpoint("Shutting down Laser Tracker.")
+        await self.scheduler_remote.cmd_powerOff.set_start()
+        await self.checkpoint("Laser Tracker turned off.")
+
```

Comment:
I wonder if we should wait for the laser to report as OFF, like we do with the power on... I think it is probably a good idea.